azext_connect/_mysql.py

```diff
@@ -54,6 +55,25 @@ def mysql_handler(resource_group, deployment_id, settings, para_dict):
     if DEFAULT_CLI.invoke(parameters):
         raise CLIError('Fail to configure SSL settings for %s.' % mysql_server_name)
 
+    # Create database, check if it exists, if not create it
+    parameters = [
+        'mysql', 'db', 'show',
+        '--name', database_name,
+        '--server-name', mysql_server_name,
+        '--resource-group', resource_group
+    ]
+    try:
+        DEFAULT_CLI.invoke(parameters)
+    except:
+        parameters = [
```

Comment:
We may discuss whether to create db. The current cupertino scope is connect existing resource. So not sure whether we need create db for users. This could be a open question.

Reply:
This supports both to create or not to, depends on if users created the resources before run "az connect".